src/boot/efi/pe.c

```diff
@@ -140,48 +141,49 @@ static size_t section_table_offset(const DosFileHeader *dos, const PeFileHeader
         return dos->ExeHeader + offsetof(PeFileHeader, OptionalHeader) + pe->FileHeader.SizeOfOptionalHeader;
 }
 
-static void locate_sections(
+static ssize_t locate_section(
```

Comment:
why not just return `const PeSectionHeader` here pointing to the right entry? seems nicer to me.